templates/step_function/backend_checks_definition.json.tpl

```diff
@@ -152,22 +145,66 @@
               "checksum.$": "$.[?(@.checksum)].checksum"
             }
           }
+        },
+        "ProcessorConfig": {
+          "Mode": "DISTRIBUTED",
+          "ExecutionType": "EXPRESS"
         }
       },
       "MaxConcurrency": 40,
       "ResultSelector": {
         "results.$": "$"
       },
-      "ItemsPath": "$.results",
-      "Next": "Redacted files",
+      "Next": "Process Map Results",
       "Catch": [
         {
           "ErrorEquals": [
             "States.ALL"
           ],
           "Next": "Notify"
         }
-      ]
+      ],
+      "Label": "Map",
+      "ItemReader": {
+        "Resource": "arn:aws:states:::s3:getObject",
+        "ReaderConfig": {
+          "InputType": "JSON"
+        },
+        "Parameters": {
+          "Bucket.$": "$.bucket",
+          "Key.$": "$.key"
+        }
+      },
+      "ResultWriter": {
+        "Resource": "arn:aws:states:::s3:putObject",
+        "Parameters": {
+          "Bucket.$": "$.bucket",
+          "Prefix.$": "$.key"
+        }
+      }
+    },
+    "Process Map Results": {
+      "Type": "Task",
+      "Resource": "arn:aws:states:::lambda:invoke",
+      "OutputPath": "$.Payload",
+      "Parameters": {
+        "Payload.$": "$",
+        "FunctionName": "${backend_checks_results_arn}"
+      },
+      "Retry": [
+        {
+          "ErrorEquals": [
+            "Lambda.ServiceException",
```

Comment:
Can these go in alphabetical order?

Reply:
I almost got all of them in alphabetical order. I've updated these.